### ISSUE TITLE
Some of the results are full of noise.

### ISSUE THREAD

**Issue by TMDTom** [Janspiry/Palette-Image-to-Image-Diffusion-Models#93]
Some of the results are full of noise. And there are  few results  successful to me. Maybe like this.
![image](https://github.com/Janspiry/Palette-Image-to-Image-Diffusion-Models/assets/89788203/7dd7bd55-ac8d-491f-aa98-f9135d5c6831)

![image](https://github.com/Janspiry/Palette-Image-to-Image-Diffusion-Models/assets/89788203/e6cae48c-b248-4073-a384-55f35fae02e8)

**Comment by SarielMa** [comment]
I had the same issue when I adopted the colorization data loader to my own MR Image translation task. When I removed the normalization in the transforms, the noise was gone. But the psnr and ssim were very poor.

**Comment by TMDTom** [comment]
> I had the same issue when I adopted the colorization data loader to my own MR Image translation task. When I removed the normalization in the transforms, the noise was gone. But the psnr and ssim were very poor.

Ok, thanks for your answer. I'll have a try on it. Normalization maybe is connected by your data distribution.

**Comment by AwayGithub** [comment]
@SarielMa @TMDTom Hi! I'm also working on the medical image dataset BraTS to perform modal translation, and the result is very poor. I'm using the colorization config file. Although the training loss is quite low (about 0.002 after 10 epoch), the validation loss is very high (about 0.80), and the result pics is very noisy. I'm wondering if and how you've solved this issue, thanks!

**Comment by AwayGithub** [comment]
@TMDTom @SarielMa Now my MRI translation task meets the same problem, around 80% of the outputs are successful, and the rest are very noisy. And this happens after I set the noise scheduler to "cosine", before that, I couldn't even get a single successful output. So did you solve the problem? And how? Thanks a lot!

**Comment by jjjje** [comment]
@AwayGithub
I also have same issue.
I trained and tested my own dataset.
But my test results are some of them were good and some of them were just noise.
My dataset was 580 size of 256*256 and I use colorization method.
I check every 1000 epoch until 5000epochs. and It was getting better but still have randomly noise results.
Did you solve it?

**Comment by TMDTom** [comment]
> [@AwayGithub](https://github.com/AwayGithub) I also have same issue. I trained and tested my own dataset. But my test results are some of them were good and some of them were just noise. My dataset was 580 size of 256*256 and I use colorization method. I check every 1000 epoch until 5000epochs. and It was getting better but still have randomly noise results. Did you solve it?

Maybe you can try the way: change the initial data distribution to 0-1 or -1-1.

**Comment by jjjje** [comment]
@TMDTom  Thank you for your answer.
I already normalize my dataset into -1~1... 
Now your experiments results didn't show noise? solve the problem? can i ask how many your train dataset?

**Comment by TMDTom** [comment]
> [@TMDTom](https://github.com/TMDTom) Thank you for your answer. I already normalize my dataset into -1~1... Now your experiments results didn't show noise? solve the problem?

I remember solving it by normalizing to either 0-1 or -1 to 1.

**Comment by jjjje** [comment]
@TMDTom Thank you.
I'm thinking of increasing my dataset..
can i ask how many was your train dataset?

**Comment by TMDTom** [comment]
> [@TMDTom](https://github.com/TMDTom) Thank you. I'm thinking of increasing my dataset.. can i ask how many was your train dataset?

My dataset contains 1,200 training images, and I trained for 100 epochs with the batch_size set to 1.
